OpenRA.Mods.Common/Traits/DamagedByTerrain.cs

```diff
@@ -62,6 +62,9 @@ public void AddedToWorld(Actor self)
 					safeTiles++;
 			}
 
+			if (totalTiles < 1)
```

Comment:
I think this would be better expressed as `== 0`, because we aren't ever worried about a negative value.